notifier/icons.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"sync"
+
+	"github.com/safing/portmaster-ui/notifier/icons"
+)
+
+var (
+	appIconEnsureOnce = new(sync.Once)
```

Comment:
There's no need to use `new()` here. The zero value of `sync.Once` is useable. Just don't copy it

Reply:
Ah, yes. I planned to reset sync.Once when it fails, but then decided not to.